plugins/obs-outputs/rtmp-stream.c

```diff
@@ -227,13 +227,6 @@ static void rtmp_stream_stop(void *data, uint64_t ts)
 	}
 }
 
-static inline void set_rtmp_str(AVal *val, const char *str)
```

Comment:
This function was added by 0cf9e0cfdd and was used in `try_connect`. Later, a commit 1ca065b0f replaced `set_rtmp_str` with `set_rtmp_dstr`.